tests/functional/example/test_speechbrain_tutorial.py

```diff
@@ -0,0 +1,159 @@
+"""Tests the minimalist example script on scitkit-learn and its integration to Oríon."""
+import json
+import os
+import subprocess
+
+import pytest
+
+import orion.core.cli
+from orion.client import create_experiment
+from orion.storage.base import setup_storage
+
+
+def json_clean(path, scale):
+    """Modifies json file to reduce it to some scale."""
+    f = open(path)
+    json_list = json.load(f)
+    new_list = {}
+    ctr = 0
+
+    for item in json_list:
+        if ctr < len(json_list) * scale:
```

Comment:
Here `json_list` is a list of examples in a dataset, right?

Reply:
Yes it is